nifi-extension-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/ElasticsearchRestProcessor.java

```diff
@@ -182,6 +186,15 @@ public interface ElasticsearchRestProcessor extends Processor, VerifiableProcess
             .required(false)
             .build();
 
+    PropertyDescriptor MAX_JSON_FIELD_STRING_LENGTH = new PropertyDescriptor.Builder()
+            .name("Max JSON Field String Length")
+            .displayName("Max JSON FieldString Length")
```

Comment:
In this case, recommend removing the `displayName` since it should be the same as the name (although there appears to be a space missing between `Field` and `String`).

Reply:
D'oh, good spot - updated